Example/Controllers/SwiftUI/Features/Settings/Features/MiniAppSettingsAccessTokenView.swift

```diff
@@ -69,6 +73,7 @@ struct MiniAppSettingsAccessTokenView: View {
             accessTokenString = tokenInfo.tokenString
             expiryDate = tokenInfo.expiryDate
         }
+        .trackPage(pageName: "Access Token")
```

Comment:
We have few localization created for all screens, like `demo.app.rat.page.name.access_token` ? Please use the same

Reply:
localization added